### PR TITLE
updated gallery.yml

### PR DESCRIPTION
**Pull request by shiftsayan**
Updated the gallery.yml file with all the projects that had been uploaded

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://shiftsayan.github.io/gci15.fossasia.org

File: _data/gallery.yml [hunk 1]
```diff
@@ -54,19 +59,69 @@
...
+
+  -
     name: Open Street Map - Postal code queries
     image: OpenStreetMap.png
     author: Duc Phan Duy
 
   -
-    name: YaCy page on ro.Wikipedia
-    image: ro-wiki-yacy.png
-    author: Dani Pop
```
Comment (jigyasa-grover) [inline _data/gallery.yml:69]: @shiftsayan Why was this removed ?

**Comment by shiftsayan** [comment]
Because that project was not in the gallery @jig08

**Comment by jigyasa-grover** [comment]
@shiftsayan Ah, yes.

@AvramPop Add your project again and don't forget to update the `index.html` file as well.

**Comment by AvramPop** [comment]
ok :)
